src/TextAnalysis.jl

```diff
@@ -60,6 +61,10 @@ module TextAnalysis
 
     export CRF, viterbi_decode, crf_loss
 
+    function __init__()
+        include(joinpath(@__DIR__, "avgPerceptronTagger_DataDeps.jl"))
```

Comment:
don't do an `include` in the `__init__`. make it a function, and call the function.

Reply:
Done.